src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Header = () => {
+  const date = new Date();
+  return (
+    <div className="headers">
+      <p className="headerp">
```

Comment:
- [ ] There should be a link from the `Detail` page to the `Home` page by clicking the "Back" button, you can implement this by using the back arrow in the Header/Navigation